src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java

```diff
@@ -26,4 +24,19 @@ public ResponseEntity<Statement> searchPscStatements(@PathVariable String compan
         Statement statement = pscStatementService.retrievePscStatementFromDb(company_number, statement_id);
         return new ResponseEntity<>(statement, HttpStatus.OK);
     }
+    /**
+     * Delete psc-statement information for a statement id.
+     *
+     * @param  statementId  the statement id to be deleted
+     * @return return 200 status with empty body
+     */
+    @DeleteMapping("/company/{company_number}/persons-with-significant-control-statement/{statement_id}/Internal")
```

Comment:
Could the internal be changed to lower case here

Reply:
Can do, it was uppercase on the JIra but I can just add a comment